ssh_key.go

```diff
@@ -9,70 +9,66 @@ import (
 
 // SSHKey represents an SSH public key, uploaded to access instances
 type SSHKey struct {
-	ID      string `json:"id"`
-	Name    string `json:"name"`
-	Region  string `json:"region"`
-	Default bool   `json:"default"`
-	CIDR    string `json:"cidr"`
-	Label   string `json:"label"`
+	ID          string `json:"id"`
+	Name        string `json:"name"`
+	FingerPrint string `json:"fingerprint"`
```

Comment:
Sorry, didn't notice this before - the English is "Fingerprint" not "FingerPrint".

Reply:
Ok, I just followed the Go standard 👍

Reply:
I'll fix it locally, but it's not Go standard. Go standard is to capitalise each word. So if the item was a "finger print" it would be "FingerPrint", and the JSON would be "finger_print", but it's all one word, so it's Fingerprint. Anyway, I've merged it now, I'll fix locally.